frontend/services/entities/scripts.ts

```diff
@@ -0,0 +1,36 @@
+import sendRequest from "services";
+import endpoints from "utilities/endpoints";
+
+export interface IScriptResult {
+  host_name: string;
+  host_id: number;
+  execution_id: number;
+  script_contents: string;
+  exit_code: number | null;
+  output: string;
+  message: string;
+  runtime: number;
+  host_timeout: boolean;
+}
+
+export default {
+  getScriptResult(id: number) {
+    const { SCRIPT_RESULT } = endpoints;
+
+    // TODO: uncomment when API is ready.
```

Comment:
remove these comments?

Reply:
@RachelElysia the API isnt quite ready so im gonna leave this in until we finish that bit.